redaxo/src/core/lib/sql/schema_dumper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * Class for generating the php code for a rex_sql_table definition.
+ *
+ * Especially useful to generate the code for the `install.php` of packages.
+ *
+ * @author gharlan
+ *
+ * @package redaxo\core
+ */
+class rex_sql_schema_dumper
+{
+    public function dump(rex_sql_table $table)
```

Comment:
wollen wir die methode in dumpTable umbennen, falls wir hier in der dumper klasse irgendwann noch was anderes dumpen können?

Reply:
alternativ, die klasse als table-dumper umbennen und dann hier dump() lassen

Reply:
z.b. Könnte man vllt zukünftig die ganze db auf einmal dumpen, oder nur ein diff zwischen 2 zuständne oder so